### DISCUSSION TITLE
Aggiornamenti

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#438] [Richieste di aiuto] [answered]
Buonasera,
è necessario installare le ultime modifiche o attendiamo il rilascio di una nuova versione di aggiornamento?
Grazie.

**Comment by trinko** [comment, marked as answer]
Se si ha necessità delle modifiche effettuate, allora è bene installare gli ultimi aggiornamenti, che sono già stati testati.
La nuova versione penso che verrà rilasciata a gennaio.

**Reply by Lucaccc** [reply]
La modifica sulla gestione delle procedure di scrutinio che cambiamenti ha apportato?

**Reply by trinko** [reply]
> La modifica sulla gestione delle procedure di scrutinio che cambiamenti ha apportato?

La modifica riguarda gli scrutini del primo trimestre/quadrimestre e del secondo trimestre (se presente): è stata aggiunto una pagina intermedia per l'inserimento/modifica delle comunicazioni dei debiti formativi.

Il resto sono correzioni varie.

**Reply by Lucaccc** [reply]
Grazie mille.
